frame/society/src/weights.rs

```diff
@@ -0,0 +1,434 @@
+	// Storage: Society Parameters (r:1 w:0)
+	fn bid() -> Weight {
+		(43_649_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(5 as Weight))
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+	}
+	// Storage: Society Bids (r:1 w:1)
+	fn unbid() -> Weight {
+		(34_775_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(1 as Weight))
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+	}
+	// Storage: Society Bids (r:1 w:1)
+	// Storage: Society Candidates (r:1 w:0)
+	// Storage: Society Members (r:2 w:1)
+	// Storage: Society SuspendedMembers (r:1 w:0)
+	fn vouch() -> Weight {
+		(34_423_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(5 as Weight))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+	}
+	// Storage: Society Bids (r:1 w:1)
+	// Storage: Society Members (r:1 w:1)
+	fn unvouch() -> Weight {
+		(24_933_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(2 as Weight))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+	}
+	// Storage: Society Candidates (r:1 w:1)
+	// Storage: Society Members (r:1 w:0)
+	// Storage: Society Votes (r:1 w:1)
+	fn vote() -> Weight {
+		(32_431_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(3 as Weight))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+	}
+	// Storage: Society Defending (r:1 w:1)
+	// Storage: Society Members (r:1 w:0)
+	// Storage: Society ChallengeRoundCount (r:1 w:0)
+	// Storage: Society DefenderVotes (r:1 w:1)
+	fn defender_vote() -> Weight {
+		(31_202_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(4 as Weight))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+	}
+	// Storage: Society Members (r:1 w:0)
+	// Storage: Society Payouts (r:1 w:1)
+	// Storage: System Account (r:1 w:1)
+	fn payout() -> Weight {
+		(44_352_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(3 as Weight))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+	}
+	// Storage: Society Members (r:1 w:1)
+	// Storage: Society Payouts (r:1 w:1)
+	fn waive_repay() -> Weight {
+		(26_934_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(2 as Weight))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+	}
+	// Storage: Society Head (r:1 w:1)
+	// Storage: Society MemberCount (r:1 w:1)
+	// Storage: Society MemberByIndex (r:0 w:1)
+	// Storage: Society Founder (r:0 w:1)
+	// Storage: Society Rules (r:0 w:1)
+	// Storage: Society Members (r:0 w:1)
+	// Storage: Society Parameters (r:0 w:1)
+	fn found_society() -> Weight {
+		(29_573_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(2 as Weight))
+			.saturating_add(T::DbWeight::get().writes(7 as Weight))
+	}
+	// Storage: Society Founder (r:1 w:1)
+	// Storage: Society MemberCount (r:1 w:1)
+	// Storage: Society Head (r:0 w:1)
+	// Storage: Society MemberByIndex (r:0 w:1)
+	// Storage: Society Rules (r:0 w:1)
+	// Storage: Society Members (r:0 w:1)
+	fn dissolve() -> Weight {
```

Comment:
a bit too good to be true. The dissolve benchmark should assume "worse case", which means there should be maximum number of votes and other data structures also around to clean.